Scrot.py

```diff
@@ -25,7 +27,10 @@
     raise Exception("'xclip' is not in $PATH.")
 
 iconPath = iconLookup("camera-photo")
-fileName = "%Y-%m-%d-%T-screenshot.png"
+screenshotDirectory = os.getenv("XDG_PICTURES_DIR", tempfile.gettempdir())
```

Comment:
In my understanding, the XDG dirs are a bit more complicated to query: The environment variables are normally not defined. They would only be defined if you sourced the `~/.config/user-dirs.dirs` in a shell.

To query the directory correctly, it looks like you have to call this command: `xdg-user-dir PICTURES` - its output is the path to use. If empty, use the temp dir. See the last lines of this link for reference: https://wiki.archlinux.org/index.php/XDG_user_directories

Then the xdg package (or xdg-user-dir?) should be added as dependency.

Reply:
ah, thanks I totally missed that..